tests/test_knowledge/test_consequents/test_takagi_sugeno_consequent.py

```diff
@@ -0,0 +1,49 @@
+from doggos.knowledge.consequents import TakagiSugenoConsequent
+import pytest
+
+
+class TestTakagiSugenoConsequent:
```

Comment:
I would like to see a test with np.ndarray passed either as function_params or input.

Reply:
I think we will need to change that to dictionaries, as we will need to find a way to align each parameter for specific LinguisticVariable in process of output calculation